cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -459,6 +459,10 @@ impl ast::UnreservedId {
                 .map(|arg| construct_method_contains_all(e, arg, loc.clone())),
             "containsAny" => extract_single_argument(args.into_iter(), "containsAny", loc)
                 .map(|arg| construct_method_contains_any(e, arg, loc.clone())),
+            "getTag" => extract_single_argument(args.into_iter(), "getTag", loc)
```

Comment:
What's the parse error like for `principal.getTag(foo)` (getting the identifier as opposed to a string)? Might be worth building in special handling for that (or opening an issue to)

Reply:
Added a test.  Out of the box, the parse error is as shown here
<img width="1106" alt="image" src="https://github.com/user-attachments/assets/b47fbe07-ceb8-491c-8193-521c04c2ce1d">

Reply:
looks nice enough. No need for any issue IMO